southpark-search/app.py

```diff
@@ -11,53 +11,84 @@
 RANDOM_SEED = 10
 random.seed(RANDOM_SEED)
 
-os.environ['REPLICAS'] = str(2)
-os.environ['SHARDS'] = str(2)
-os.environ['TMP_DATA_DIR'] = '/tmp/jina/southpark'
-os.environ['JINA_PORT'] = str(45678)
-
 
 def get_random_ws(workspace_path, length=8):
     letters = string.ascii_lowercase
     dn = ''.join(random.choice(letters) for i in range(length))
     return os.path.join(workspace_path, dn)
 
-
 def print_topk(resp, word):
     for d in resp.search.docs:
         print(f'Ta-Dah🔮, here are what we found for: {word}')
-        for idx, kk in enumerate(d.topk_results):
-            score = kk.score.value
+        for idx, match in enumerate(d.matches):
+            score = match.score.value
             if score < 0.0:
                 continue
-            doc = kk.match_doc.text
+            doc = match.meta_info.decode()
```

Comment:
same here. Has it been tested? I think is never filled the `meta_info` field I think.